src/test-c-gen/function_return_builder.cpp

```diff
@@ -0,0 +1,203 @@
+#include "function_return_builder.h"
+
+#include <test-c-gen/expr2cleanc.h>
+
+/*******************************************************************\
+Function: function_return_buildert::function_return_buildert
+Inputs:
+ all_inputs - All the inputs from the interpreter
```

Comment:
Sorry, I'm not sure I follow. As in the comment for the constructor should explain all input-y functions within the class?